extensions/amp-iframe/validator-amp-iframe.protoascii

```diff
@@ -31,6 +31,10 @@ tags: {  # <amp-iframe>
   tag_name: "AMP-IFRAME"
   requires_extension: "amp-iframe"
   attrs: {
+    name: "allow"
+    value_regex: "geolocation|fullscreen|payment|transparency"
```

Comment:
@honeybadgerdontcare @Gregable runtime was not updated to support `allow` I am sending a PR soon. My question is whether this regex is correct, multiple values can be set for allow (e.g. `geolocation; microphone`) also these values can change quickly (`microphone` was already added as well). 

so can we go with not-validating the actual value of this attribute?